src/brasil/gov/portlets/i18n.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
we have a helper so we don't have to include this all over the packages: https://github.com/collective/buildout.plonetest/blob/master/templates/rebuild_i18n.sh.in

analyze it to see if it can be improved.